### ISSUE TITLE
\' in an html template throws a syntax error

### ISSUE THREAD

**Issue by joechrz** [bclinkinbeard/partialify#10]
Since angular (and some other templating langs) allow some degree of inline scripting it can be necessary to escape content.  Due to the way partialify inlines the template, this can cause a syntax error:

```
<!DOCTYPE html>
<html>
<head>
  <title></title>
</head>
<body>

<span ng-class="{ 'foo': 'foo\'s bar' }">
    Text in HTML
</span>

</body>
</html>
```

Partialify escapes the single quote, but since it's preceeded by a backslash, the backslash ends up being escaped.

```
module.exports = '<!DOCTYPE html>\n<html>\n<head>\n <title></title>\n</head>\n<body>\n\n<span ng-class="{ \'foo\': \'foo\\'s bar\' }">\n Text in HTML....
```

**Comment by srivathsah** [comment]
+1

**Comment by srivathsah** [comment]
another case whre we need \ is in place holders for eg: domain\username

**Comment by csbun** [comment]
sorry, when i install from npm

```
npm i partialify@3.1.5
```

i still get this in `custom.js`

```javascript
// line 27 ...
        this.queue(buffer); // prevent "double" transforms
```

**Comment by joechrz** [comment]
@csbun this fix was reverted in 9767b866bcb97b82d58c912eb0846e02bf20241b due to an issue it caused with resrcify
